Python/Discovering Subdomains/program.py

```diff
@@ -0,0 +1,29 @@
+import requests
+
```

Comment:
The code looks clean. As an enhancement, you can wrap the working into a `user defined functions` and use `if __name__== __main__:` as a starting point of the program. Would be great if you can address this.